src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfigTest.java

```diff
@@ -46,4 +50,39 @@ public void testSetHttpTimeoutsConfig() {
         (Integer) 15000
     );
   }
+
+  @Test
+  public void testSslConfigs() {
+    props.put("elastic.https.ssl.keystore.location", "/path");
+    props.put("elastic.https.ssl.keystore.password", "opensesame");
+    props.put("elastic.https.ssl.truststore.location", "/path2");
+    props.put("elastic.https.ssl.truststore.password", "opensesame2");
+    ElasticsearchSinkConnectorConfig config = new ElasticsearchSinkConnectorConfig(props);
+    Map<String, Object> sslConfigs = config.sslConfigs();
+    Assert.assertTrue(sslConfigs.size() > 0);
+    Assert.assertEquals(
+        new Password("opensesame"),
+        sslConfigs.get("ssl.keystore.password"));
```

Comment:
Nit: this line and several others doesn't match our style of putting the closing `);` on the next line.